### PR TITLE
release/1.3.3

### PR DESCRIPTION
**Pull request by fmauch**
As Always I will rebase-merge this once approved and push the tag manually.

### CODE REVIEW

**Review by urmahp — COMMENTED** [review]
Overall it looks good to me. One comment regarding clarification for "added support for timeout and max_num_tries"

Will #175 be part of this release? If yes then we need to add that to the changelog.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,29 @@
 Changelog for package ur_client_library
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+1.3.3 (2023-09-05)
+------------------
+*  Add support for setting socket max num tries and reconnect timeout (`#172 <https://github.com/UniversalRobots/Universal_Robots_Client_Library/issues/172>`_)
+* Unify socket open (`#174 <https://github.com/UniversalRobots/Universal_Robots_Client_Library/issues/174>`_)
+* added support for timeout and max_num_tries
```
[the remark this answers is not in the document]
Comment (fmauch) [reply CHANGELOG.rst:9]: Hm, that seems to be a duplicate from #172 to be honest. Maybe let's remove that line.

[the remark this answers is not in the document]
Comment (fmauch) [reply CHANGELOG.rst]: I clarified the changelog a bit.

**Review by urmahp — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmauch) [reply CHANGELOG.rst]: I clarified the changelog a bit. #175 is not necessarily a part of this release.

**Pull request merged**